prime-angular-frontend/src/app/modules/paper-enrolment/pages/care-setting-page/care-setting-form-state.class.ts

```diff
@@ -94,14 +94,6 @@ export class CareSettingFormState extends AbstractFormState<CareSettingForm> {
     return this.fb.control(checkState);
   }
 
-  public disableCareSetting(careSettingCode: number): boolean {
```

Comment:
Needs to go back in and prevent use of device provider in production

Reply:
Disabled